cluster/service/cluster_blackbox_test.go

```diff
@@ -126,24 +132,24 @@ func updateClusterConfigFile(t *testing.T, to, from string) {
 func waitForConfigUpdate(t *testing.T, config *configuration.ConfigurationData, expected bool) {
 	for i := 0; i < 30; i++ {
 		time.Sleep(100 * time.Millisecond)
-		cluster := config.GetOSOClusterByURL("https://api.starter-us-east-2a.openshift.com")
-		require.NotNil(t, cluster)
-		if expected == cluster.CapacityExhausted {
+		c := config.GetClusterByURL("https://api.starter-us-east-2a.openshift.com")
```

Comment:
Also check if the type is set. We need that to make sure we set the default type when updating by watcher even if it’s not set in the config file.
This is covered for initial config load but I also want to cover it for the watche.

Reply:
done in 3c92448